projects/ngrx.io/content/guide/store/selectors.md

```diff
@@ -333,7 +333,7 @@ export const selectLastStateTransitions = (count: number) => {
     select(selectProjectedValues),
     // Combines the last `count` state values in array
     scan((acc, curr) => {
-      return [ curr, acc[0], acc[1] ].filter(val => val !== undefined);
+      return [ curr, acc[0], acc[1] ].filter(val => val !== undefined).slice(0,count);
```

Comment:
This won't be enough because `acc[0], acc[1]` is still hard coded
```suggestion
      return [ curr, ...acc ].filter((val, index) => index < count && val !== undefined)
```

Reply:
Wow... not sure how I missed that.

Reply:
I committed your suggestion but somehow managed to do it before changing the commit message... not sure if this is going to be an issue as it doesn't follow the guidelines?

Reply:
It's good, we'll squash the commits together when we merge.